seedlet/inc/headstart/en.json

```
@@ -60,12 +46,12 @@
             "hs_custom_meta": "_hs_first_post",
             "hs_sharing": 1,
             "hs_like_status": 1,
-            "attachment_url": "https:\/\/headstartdata.files.wordpress.com\/2016\/06\/pexels-photo-30732.jpg",
+            "attachment_url": "https://seedletdemo.files.wordpress.com/2020/07/envelope.png",
```

```suggestion
            "attachment_url": "https://headstartdata.files.wordpress.com/2020/07/envelope.jpg",
```

I've uploaded a higher-res version of this to the headstartdata site.